core/src/main/scala/com/lightbend/paradox/markdown/Writer.scala

```diff
@@ -92,6 +93,10 @@ object Writer {
 
   val DefaultSourceSuffix = ".md"
   val DefaultTargetSuffix = ".html"
+  val DefaultIllegalLinkPath =
+    // #DefaultIllegalLinkPath
+    raw"""^(?!https?:).*\.md(#.*)?""".r
```

Comment:
Why the `^(?!https?:).*`?

Reply:
The problematic links are the local links within the Paradox generated docs. External links may point to eg. Github .md files such as https://github.com/lightbend/config/blob/master/README.md#using-the-library

Reply:
aaah I misread the regex :smile: :+1: